modules/AlphaWalletFoundation/AlphaWalletFoundation/Extensions/Publishers/Promise+Publishers.swift

```diff
@@ -100,3 +100,33 @@ extension Publisher {
         }
     }
 }
+
+public extension Promise {
```

Comment:
Good to add a TODO to the extension like this:

```
//TODO this exist for migration from `Promise`/`Future` to async-await. We can remove this once all usage have been migrated.
```

Same for the one for `Future`